tests/closure/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -108,11 +108,9 @@ $longArgs_shortVars = function (
     // body
 };
 
-$shortArgs_longVars = function ($arg) use (
-    $longVar1,
-    $longerVar2,
-    $muchLongerVar3
-) {
+$shortArgs_longVars = function (
```

Comment:
I'm unsure on this one - but I think the larger problem is more important. If we think the original formatting is better, we'll probably have to set up a `conditionalGroup` for functions, figuring out the order of when to break the function arguments vs the group arguments

Reply:
Totally agree. :+1: